### DISCUSSION TITLE
Crank it

### DISCUSSION THREAD

**Discussion by LouderthanThunder** [Lambosaurus/hives-and-torches#38] [Feats]
Crossbow gains advantage. On a crit fail the string breaks

**Comment by Lambosaurus** [comment]
Critical failures rules should only really be invoked on rules that give disadvantage.
This is because you can **only** critical fail when you have disadvantage with current rules.

**Reply by LouderthanThunder** [reply]
It would be nice for some kind of crit fail for risk/reward

**Reply by Lambosaurus** [reply]
Well, it could add disadvantage, and then give you flat bonus

**Reply by Lambosaurus** [reply]
But then the user would just save this feature for when they had at least `x1` advantage....
Idk, im open to ideas - it just needs to be reworked a bit.

**Reply by LouderthanThunder** [reply]
Cool cool. I like the oh fuck momen 
T when your crossbow becomes useless
